perseo-main/pom.xml

```diff
@@ -78,6 +78,16 @@
             <groupId>com.telefonica.iot</groupId>
             <version>1.5.0</version>
         </dependency>
+        <dependency>
```

Comment:
You explained why org.owasp.encoder dependency is needed at https://github.com/telefonicaid/perseo-core/pull/151#discussion_r358836399. However, what about org.json? Is this really needed (not sure, but it seem that in the first version of the PR this dependency didn't was included)?

Reply:
We included this dependency because we detected that the package org.json was included directly in the packages of the project. This dependency was outdated and also generated a number of issues (37). 
All of them were solved when we added the updated dependency through Maven.

Reply:
We included this dependency because we detected that the package org.json was included directly in the packages of the project. This dependency was outdated and also generated a number of issues (37).
All of them were solved when we added the updated dependency through Maven.

Reply:
Well done! Better to use existing jar files in maven than having the source code of a third-party library in the repository itself.

NTC